### PR TITLE
Chore/use dids

### PR DESCRIPTION
**Pull request by Magnus-Kuhn**

### CODE REVIEW

**Comment by Magnus-Kuhn** [comment]
The address generation algorithm deviates in the hex strings case from Confluence - I checked it stepping through the runtime code

File: _docs_explore/60-addresses.md [hunk 1]
```diff
@@ -3,48 +3,50 @@ title: "enmeshed Addresses"
 permalink: /explore/addresses
 ---
 
+<!-- TODO: consider new enmeshed key types  -->
```
Comment (jkoenig134) [inline _docs_explore/60-addresses.md:6]: a TODO in an open PR, can you resolve this?

File: _docs_integrate/integration-example.md [hunk 2]
```diff
@@ -236,7 +237,7 @@ Example:
         "id": "RELmJj25x2bZW0VXzAiQ",
         ...
         "status": "Pending",
-        "peer": "id19Sy75wjCWhQSxsbMiGLn6iSBfWvQmot5b",
+        "peer": "did:e:prod.enmeshed.eu:dids:60326ff5075e0d7378990c",
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:240]:
```suggestion
        "peer": "did:e:_________________",
```

File: _docs_integrate/integration-example.md [hunk 3]
```diff
@@ -79,7 +79,8 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD`.
+You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:stage.enmeshed.eu:dids:54013af2a3bffe9dc8cb99`.
```
Comment (jkoenig134) [inline _docs_integrate/integration-example.md:82]: you have referenced this address, but not updated the connectors baseurl

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

File: _includes/rapidoc
Comment (jkoenig134) [inline _includes/rapidoc:1]: please undo formatting here, your formatter destroyed that file :/

Comment (jkoenig134) [reply _includes/rapidoc:1]: `F1 -> save without formatting` is your friend for these include files.

**Review by jkoenig134 — APPROVED** [review]
LGTM. Make sure to use dids in all other PRs too.

**Pull request merged**